project/CLI.py

```diff
@@ -132,7 +133,7 @@ def test_add_team_not_gm(self):
         self.assertEqual("Permission denied", self.cli.command("addTeam Team1 1234"), "Only game maker can add teams")
 
     def test_add_team_duplicate(self):
-        self.assertEqual("Added team", self.cli.command("addTeam Team1 1234"), "Failed to add team")
+        self.assertEqual("Team added", self.cli.command("addTeam Team1 1234"), "Failed to add team")
```

Comment:
We should move this stuff to a String Constant file so we don't keep making mistakes like this.